### ISSUE TITLE
Defining Progress Spinner Height and Width

### ISSUE THREAD

**Issue by walioul** [Nikoloutsos/KLoadingSpin#3]
Hello Boss,

I used your nice library KLoadingSpin . I want to make Progress Spinner smaller than default size. How can I set it's height and width please ?

Advance thanks

**Comment by Nikoloutsos** [comment]
Hey walioul,
are you referring to circle's size?

If yes then I have not added a method for doing so..
But I will make a new release version the upcoming week and I will add this, so I will reply to your question then with the code you need to achieve what you're looking for

Have a nice day!

**Comment by walioul** [comment]
Hello,
Yes I am referring to circle's size. I am waiting for your next release.

Thank you very much

**Comment by Nikoloutsos** [comment]
**Here is an example picture:** 
![](https://i.imgur.com/nfDLOrK.png)

**How to achieve this result:**
 ```
  // For showing
  KLoadingSpin a = findViewById(R.id.KLoadingSpin);
  a.changeSpinnerDimension(100, 10); // Available in 1.1+ version.
  a.startAnimation();
  a.setIsVisible(true);
```

```
dependencies {
    implementation 'com.github.Nikoloutsos:KLoadingSpin:1.1.0'
}

```
**Explanation of changeSpinnerDimension():**
It takes 2 paramaters:

1. The size of the circle. (The one you asked for)
2. The fatness of the ring. (In the above picture I've used 10)

I hope walioul that this method answers your question!
